nexus/src/nexus.rs

```diff
@@ -1732,6 +1734,23 @@ impl Nexus {
         let system_router_id = Uuid::new_v4();
         let default_route_id = Uuid::new_v4();
         let default_subnet_id = Uuid::new_v4();
+
```

Comment:
Note: prior to this change, we actually created the system router for the VPC in the database _before_ the VPC itself.  This change creates the VPC first.  Both seem invalid in that each creates a record in the database that refers to another record that doesn't exist (usually for a brief period).  I didn't think this was any worse, and it made the authz implementation simpler (because we assume the VPC exists in order to do an authz check on creating a router inside it).

There are already multiple TODOs in this function to use sagas or a transaction instead.